axelor-prestashop/src/main/resources/i18n/messages_fr.csv

```diff
@@ -1,43 +1,102 @@
 "key","message","comment","context"
+"<p>WARNING: Order has been modified on PrestaShop but could not be updated locally.</p>","<p>ATTENTION : la commande a été modifiée sur PrestaShop mais les modifications n'ont pas pu être répercutées.</p>",,
 "Action",,,
-"App prestashop","App Prestashop",,
-"Awaiting Cash On Delivery validation",,,
-"Awaiting bank wire payment",,,
-"Awaiting check payment",,,
-"Batchs",,,
-"Canceled",,,
+"An HTTP error occured while checking access rights: %s","Une erreur est survenue lors de la vérification des permissions : %s",,
+"An error occured while checking Prestashop access rights: %s, see server logs for details","Une erreur est survenue lors de la vérification des permissions : %s, consultez les fichiers journaux pour plus de détails",,
+"Anomalies",,,
+"App prestashop",,,
+"Axelor texts languages (for descriptions of products, name of countries, aso.)","Langue des textes sous Axelor (description des produits, noms des pays, etc.)",,
+"Batch","Traitement par lot",,
+"Batch %s unknown","Traitement par lot %s inconnu",,
+"Batches","Traitements par lot",,
 "Code",,,
-"Company",,,
-"Created by",,,
-"Created on",,,
+"Company","Raison sociale",,
+"Connection","Connexion",,
+"Connection successful","Connexion établie avec succès",,
+"Countries are handled on prestashop","Les pays sont gérées sous PrestaShop",,
+"Created by","Créé par",,
```

Comment:
Here is better to keep this kind of translation empty. Thereby, if we want update this translation, we have to do it on the first declaration only (on ADK here)

Reply:
Removed those having a match in another module

Reply:
I think there are some similar kind of translations yet : 
- Information
- Delivered
- Batch
- Batches
- Connection
- Name
- Invoiced
- Payment Mode

Reply:
Fixed, I let "connection" since I didn't find it in another messages*.csv